src/main/kotlin/com/mparticle/kits/GoogleAnalyticsFirebaseGA4Kit.kt

```diff
@@ -733,6 +851,19 @@ class GoogleAnalyticsFirebaseGA4Kit : KitIntegration(), KitIntegration.EventList
         private const val CURRENCY_FIELD_NOT_SET =
             "Currency field required by Firebase was not set, defaulting to 'USD'"
         private const val USD = "USD"
+        //Constants for Read Consent
+        private const val consentMappingSDK = "consentMappingSDK"
+        enum class GoogleConsentValues(val values: String) {
```

Comment:
The parameter here is called `values` but it seems it's only a single value. Could this be renamed to `value` to be more clear? With `values` it makes me expect an array or something.